### ISSUE TITLE
live versions chapters links are all broken

### ISSUE THREAD

**Issue by nrnoble** [meteorinaction/ch05-fullyreactivediting#1]
example: http://mia-chapter5.meteor.com/.

Our course at Green River College is using this book. It would great if the live links worked so we can see working examples.

**Comment by yauh** [comment]
I am truly sorry for the broken links. Unfortunately the MDG decided to kill off their free hosting and we haven't gotten around to finding another host for the examples yet. If you have any suggestions, I am open to it :)

**Comment by nrnoble** [comment]
Thanks. I don't mind downloading and running the book's examples on local system. 

Can I run multiple meteor applications (book examples) by simply changing the port: --Port 3001 for each application?

**Comment by yauh** [comment]
Absolutely! As long as you remember to reserve the ports for MongoDB. For example, if you run an app on port 3000 it will automatically run MongoDB on port 3001. So you can run meteor apps on the even ports and have room for the database on the odd ports ;)

**Comment by nrnoble** [comment]
thanks for the info.  Works great. http://meteor.nealnoble.net:3002 (chapter 4 house-sitter example)
